src/index.ts

```diff
@@ -0,0 +1,26 @@
+import { connectDB } from "./database/databaseConnect";
+import { router } from "./routes/usersRoutes";
+
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import * as dotenv from "dotenv";
+dotenv.config();
+const app = express();
+app.use(bodyParser.json());
+const localAddress = "http://localhost:3000";
```

Comment:
I think you can have constants file and movie `const localAddress = "http://localhost:3000";` this to there